### ISSUE TITLE
Azshara shredder quests.

### ISSUE THREAD

**Issue by Evilwowshaman** [GuabinaCore/WoWKoi#702]
In the inital Azshara questline where you need to steal a Shredder, you learn every single ability the shredder can use as you get on to it.

On retail, it learns the abilities as you get further in to the quest.

**Comment by Evilwowshaman** [comment]
To add in, the healing ability of the shredder does not work. It just states that you're "Immune" whenever you use it.

**Comment by Arkanheal** [comment]
Azshara isn't fully scripted yet, you shouldn't go there  :)

**Comment by Spydoc** [comment]
Another thing to add to this report. If you mount a shredder and then dismount it, you are not able to mount a new shredder for the entirety of the quest chain. This is what I did to reproduce this problem:
1. Completed the first two quests with the shredder normally.
2. Upon taking the 3rd quest I dismounted the shredder.
3. Upon taking down any shredder to low enough health in order for it to become friendly, the cursor didn't turn into the mount up icon.
